recipes/mgs/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  0.1.2:
+    url: https://github.com/theodelrieu/mgs/releases/download/v0.1.2/mgs.zip
```

Comment:
not very important but https://github.com/theodelrieu/mgs/archive/v0.1.2.tar.gz is twice smaller

Reply:
The `.tar.gz` (repository source code) is split in several conan packages (I use Conan workspace to develop). It's automatically created by the GitHub action.

The `mgs.zip` contains the full source code and is way easier to package (the extra size is because of te `xxx-config.cmake` files.